pkg/apis/v1beta1/labels.go

```diff
@@ -121,6 +121,7 @@ var (
 	AnnotationEC2NodeClassHash                = apis.Group + "/ec2nodeclass-hash"
 	AnnotationEC2NodeClassHashVersion         = apis.Group + "/ec2nodeclass-hash-version"
 	AnnotationInstanceTagged                  = apis.Group + "/tagged"
+	AnnotationAMIVersionCompatibility         = apis.CompatibilityGroup + "/v1-ami-version-conversion"
```

Comment:
This is the label that's applied to the v1beta1 NodeClass which is why it's here. We discussed this offline though, yet to see if it's in this review, but you're right that we can just drop this label since we know the storage version is v1.